src/test/regress/expected/multi_router_planner.out

```diff
@@ -774,15 +774,64 @@ SELECT a.author_id as first_author, b.word_count as second_word_count
 	FROM articles_hash a, articles_single_shard_hash b
 	WHERE a.author_id = 10 and a.author_id = b.author_id
 	ORDER BY 1,2 LIMIT 3;
-DEBUG:  Creating router plan
-DEBUG:  query has a single distribution column value: 10
- first_author | second_word_count
```

Comment:
@JelteF scenarios like this can change behavior coz the single shard is now on a different node

Reply:
I don't think that's a big issue (although we should modify the test accordingly).

The reason I don't think it's a big issue is that it requires setting `citus.enable_non_colocated_router_query_pushdown`. Which already is a not-recommended GUC, because it basically requires that you don't ever do re-balancing.